fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluids/v1/math/Drops.java

```diff
@@ -0,0 +1,73 @@
+package net.fabricmc.fabric.api.fluids.v1.math;
+
+import net.minecraft.nbt.CompoundTag;
+
+/**
+ * Drops are the standard unit of fluid for the fluid api.
+ * 1/{@link #BUCKET} is the smallest amount of fluid that can be described with the Drops system.
+ * Stored in a long, at one bucket per tick, would take around 11.6 million years to overflow!
+ */
+public class Drops {
+	/**
+	 * because of constant inlining use {@link #getBucket()} and {@link #getBuckets(long)}.
+	 */
+	private static final long BUCKET = 2520L /*this number is divisible by all numbers from 1 to 9 inclusive.*/;
```

Comment:
Maybe add a @Deprecated tag here just in case?

Reply:
seems unnecessary, the field is private after all